__tests__/get-requests.test.js

```diff
@@ -63,16 +64,15 @@ describe("/api/articles", () => {
     return request(app)
       .get("/api/articles")
       .expect(200)
-      .then(({ body: articles }) => {
-        expect(articles[0].article_id).toBe(3);
-        expect(articles.length).toBe(12);
+      .then(({ body: {articles} }) => {
+        expect(articles).toBeSorted()
```

Comment:
it'd be better to specify what we're sorting by here eg. .toBeSortedBy("created_at", {descending:true} would make sure it's the correct sort query, you might need to use it later when we're sorting by different queries. The default .toSortedBy() just checks if the array is sorted in ascending order which isn't what we want